src/logic/documents/scorecards.js

```diff
@@ -360,6 +368,37 @@ const scorecard = ({
   } = scorecardPaperSizeInfos[scorecardPaperSize];
   const scorecardWidth = pageWidth / scorecardsPerRow - 2 * horizontalMargin;
 
+  let printSecondScramblerBox = false;
+  if (
+    printSecondScramblerForTopRankedCompetitors ||
+    printSecondScramblerForFinalRounds
+  ) {
```

Comment:
We don't need this outer conditional, each of the inner conditionals already checks the respective value.